packages/pwa-kit-react-sdk/src/ssr/server/express.js

```diff
@@ -203,6 +204,10 @@ export const createApp = (options) => {
     // because it's an origin, it does not end with a slash.
     options.appOrigin = process.env.APP_ORIGIN = `${options.protocol}://${options.appHostname}`
 
+    // Make the config available in a isomorphic scope. Preventing us from
+    // having to use webpack externals when bundling the `cosmiconfig` library.
+    setConfig(options.mobify)
+
```

Comment:
The call to `updatePackageMobify` mutates this config object. Wouldn't you want to do `updatePackageMobify` _before_ doing `setConfig`?

Reply:
After inspecting don't believe that function is doing any mutations.. but it's probably wise to order it in the way you suggested.